src/Builder/Pdf/AbstractChromiumPdfBuilder.php

```diff
@@ -39,6 +71,27 @@ public function setConfigurations(array $configurations): static
         return $this;
     }
 
+    /**
+     * @param list<Cookie|array{name: string, value: string, domain: string, path?: string|null, secure?: bool|null, httpOnly?: bool|null, sameSite?: 'Strict'|'Lax'|null}> $cookies
+     */
+    public function cookies(array $cookies): static
```

Comment:
```suggestion
    public function cookies(...$cookies): static
```
Like `assets(string ...$assets)`